src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final int amount = 100;
```

Comment:
Use capital letters for constant names.
Use more informative names.
```suggestion
    private static final int MAX_NUMBER = 100;
```